doc/source/whatsnew/v1.2.0.rst

```diff
@@ -249,7 +249,7 @@ Datetimelike
 - Bug in :class:`DateOffset` where attributes reconstructed from pickle files differ from original objects when input values exceed normal ranges (e.g months=12) (:issue:`34511`)
 - Bug in :meth:`DatetimeIndex.get_slice_bound` where ``datetime.date`` objects were not accepted or naive :class:`Timestamp` with a tz-aware :class:`DatetimeIndex` (:issue:`35690`)
 - Bug in :meth:`DatetimeIndex.slice_locs` where ``datetime.date`` objects were not accepted (:issue:`34077`)
-- Bug in :meth:`DatetimeIndex.searchsorted`, :meth:`TimedeltaIndex.searchsorted`, :meth:`PeriodIndex.searchsorted`, and :meth:`Series.searchsorted` with ``datetime64``, ``timedelta64`` or ``Period`` dtype placement of ``NaT`` values being inconsistent with ``NumPy`` (:issue:`36176`,:issue:`36254`)
```

Comment:
can you move notes in the Other section if possible

Reply:
Why is "Datetimelike" not the appropriate section here?

Reply:
I think what is being asked is orthogonal to the changes in this PR. will look soon.

Reply:
> can you move notes in the Other section if possible (to a more relevant section)

doesn't seem possible with clarity.

Reply:
this was a general request, nothing to do with this PR really. but since you are changing things anyways in the whatsnew it would make sense. but can do later too.